<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatureViewer</class>
 <widget class="QMainWindow" name="CreatureViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Creature Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../../core/ressources/ressources.qrc">
    <normaloff>:/img/icons/entity</normaloff>:/img/icons/entity</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuAdd_Creature">
     <property name="title">
      <string>Add creature</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../core/ressources/ressources.qrc">
       <normaloff>:/img/icons/entity_add</normaloff>:/img/icons/entity_add</iconset>
     </property>
     <addaction name="actionSpider"/>
    </widget>
    <addaction name="actionNew_creature"/>
    <addaction name="separator"/>
    <addaction name="actionSave_selected_creature"/>
    <addaction name="menuAdd_Creature"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_CreatureViewer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QDockWidget" name="dwCreature">
   <property name="minimumSize">
    <size>
     <width>380</width>
     <height>300</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="../../../../core/ressources/ressources.qrc">
     <normaloff>:/img/icons/entity</normaloff>:/img/icons/entity</iconset>
   </property>
   <property name="windowTitle">
    <string>Creature</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dwInspector">
   <property name="minimumSize">
    <size>
     <width>400</width>
     <height>200</height>
    </size>
   </property>
   <property name="windowIcon">
    <iconset resource="../../../../core/ressources/ressources.qrc">
     <normaloff>:/img/icons/inspector</normaloff>:/img/icons/inspector</iconset>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>Inspector</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <action name="actionSave_selected_creature">
   <property name="icon">
    <iconset resource="../../../../core/ressources/ressources.qrc">
     <normaloff>:/img/icons/entity_save</normaloff>:/img/icons/entity_save</iconset>
   </property>
   <property name="text">
    <string>Save selected creature</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../../../core/ressources/ressources.qrc">
     <normaloff>:/img/icons/close</normaloff>:/img/icons/close</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionLoad_Creature">
   <property name="text">
    <string>Load creature</string>
   </property>
  </action>
  <action name="actionSpider">
   <property name="text">
    <string>Spider</string>
   </property>
  </action>
  <action name="actionNew_creature">
   <property name="icon">
    <iconset resource="../../../../core/ressources/ressources.qrc">
     <normaloff>:/img/icons/entity_add</normaloff>:/img/icons/entity_add</iconset>
   </property>
   <property name="text">
    <string>New creature</string>
   </property>
  </action>
  <action name="actionAbout_CreatureViewer">
   <property name="icon">
    <iconset resource="../../../../core/ressources/ressources.qrc">
     <normaloff>:/img/icons/help</normaloff>:/img/icons/help</iconset>
   </property>
   <property name="text">
    <string>About CreatureViewer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../../../core/ressources/ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>CreatureViewer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
